db/migrate/20170217194936_add_properties_to_physical_servers.rb

```diff
@@ -0,0 +1,16 @@
+class AddPropertiesToPhysicalServers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :physical_servers, :hostname, :string
+    add_column :physical_servers, :ipv_4_addresses, :string, :array => true, :default => []
+    add_column :physical_servers, :ipv_6_addresses, :string, :array => true, :default => []
+    add_column :physical_servers, :mac_addresses, :string, :array => true, :default => []
```

Comment:
I think that @bdunne and @Fryguy are right about this ... This should leverage `computer_systems`, `hardwares`, `guest_devices`, and `networks` tables.

And, then the `firmwares` from above should fit into that mix appropriately.  Probably as either just fields in `hardwares`, or as a table hanging off of `hardwares`.

There are still some open questions to figure out:

1. What is a firmware?  Or, maybe a better question is, do we only have one per `PhysicalServer`?  In other words, does each network card have a firmware?  Or, is this just the firmware for the motherboard?

2. Does a nic/port have a UUID in the xclarity data?  When the nic details are stored in the `guest_devices` table, what should go in the `guest_devices.uid_ems` field?  For instance, for vmware hosts, the `uid_ems` field is populated with the device "name".